Scripts/WikiUpdater.cs

```diff
@@ -0,0 +1,379 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading;
+using System.Threading.Tasks;
+using AngleSharp.Html.Dom;
+using Scripts;
+using ScriptsBase.Models;
+using ScriptsBase.Utilities;
+
+public static class WikiUpdater
+{
+    private const string ORGANELLE_CATEGORY = "https://wiki.revolutionarygamesstudio.com/wiki/Category:Organelles";
+    private const string WIKI_FILE = "simulation_parameters/common/wiki.json";
+    private const string ENGLISH_TRANSLATION_FILE = "locale/en.po";
+
+    /// <summary>
+    ///   Compounds to replace with custom BBCode when appearing in bold on wiki pages.
+    /// </summary>
+    private static readonly string[] CustomBbcodeCompounds =
```

Comment:
Would it be possible to automatically detect this information by reading the compounds.json / en.po to find the English text for the compound names?

Reply:
I considered this and agree it's a better long-term solution, but decided not to for now to avoid bloating this PR even more. If you feel it's important I can add it.

Reply:
I won't require it, but at least a TODO comment would be nice. I worry though that by the time we ever add a new compound it's going to be a case of no one remembering this and that one compound being "bugged" in the thriveopedia for a while as no one notices.

Reply:
Added. I'll take a look at implementing this after this PR.